pkg/cluster/deploybaseresources_additional.go

```diff
@@ -499,6 +487,15 @@ func (m *manager) networkPublicLoadBalancer(azureRegion string) *arm.Resource {
 	}
 
 	if m.doc.OpenShiftCluster.Properties.APIServerProfile.Visibility == api.VisibilityPublic {
```

Comment:
bug? We are now only conditionally adding `public-lb-ip-v4` FIP config and outbound rule on Public clusters. Before, it appears we did this for public and private. Is this intentional?

Reply:
This is intentional.  My thought behind this is when customer provided outbound IPs/Prefixes are being used this IP would be deleted if it does not have a load balancing rule associated with it.